privaterelay/models.py

```diff
@@ -0,0 +1,561 @@
+
+
+class Profile(models.Model):
+    user = models.OneToOneField(User, on_delete=models.CASCADE)
+    api_token = models.UUIDField(default=uuid.uuid4)
+    num_address_deleted = models.PositiveIntegerField(default=0)
+    date_subscribed = models.DateTimeField(blank=True, null=True)
+    date_subscribed_phone = models.DateTimeField(blank=True, null=True)
+    # TODO MPP-2972: delete date_phone_subscription_checked in favor of
+    # date_phone_subscription_next_reset
+    date_phone_subscription_checked = models.DateTimeField(blank=True, null=True)
+    date_phone_subscription_start = models.DateTimeField(blank=True, null=True)
+    date_phone_subscription_reset = models.DateTimeField(blank=True, null=True)
+    date_phone_subscription_end = models.DateTimeField(blank=True, null=True)
+    address_last_deleted = models.DateTimeField(blank=True, null=True, db_index=True)
+    last_soft_bounce = models.DateTimeField(blank=True, null=True, db_index=True)
+    last_hard_bounce = models.DateTimeField(blank=True, null=True, db_index=True)
+    last_account_flagged = models.DateTimeField(blank=True, null=True, db_index=True)
+    num_deleted_relay_addresses = models.PositiveIntegerField(default=0)
+    num_deleted_domain_addresses = models.PositiveIntegerField(default=0)
+    num_email_forwarded_in_deleted_address = models.PositiveIntegerField(default=0)
+    num_email_blocked_in_deleted_address = models.PositiveIntegerField(default=0)
+    num_level_one_trackers_blocked_in_deleted_address = models.PositiveIntegerField(
+        default=0, null=True
+    )
+    num_email_replied_in_deleted_address = models.PositiveIntegerField(default=0)
+    num_email_spam_in_deleted_address = models.PositiveIntegerField(default=0)
+    subdomain = models.CharField(
+        blank=True,
+        null=True,
+        unique=True,
+        max_length=63,
+        db_index=True,
+        validators=[valid_available_subdomain],
+    )
+    # Whether we store the user's alias labels in the server
+    server_storage = models.BooleanField(default=True)
+    # Whether we store the caller/sender log for the user's relay number
+    store_phone_log = models.BooleanField(default=True)
+    # TODO: Data migration to set null to false
+    # TODO: Schema migration to remove null=True
+    remove_level_one_email_trackers = models.BooleanField(null=True, default=False)
+    onboarding_state = models.PositiveIntegerField(default=0)
+    onboarding_free_state = models.PositiveIntegerField(default=0)
+    auto_block_spam = models.BooleanField(default=False)
+    forwarded_first_reply = models.BooleanField(default=False)
+    # Empty string means the profile was created through relying party flow
+    created_by = models.CharField(blank=True, null=True, max_length=63)
+    sent_welcome_email = models.BooleanField(default=False)
+    last_engagement = models.DateTimeField(blank=True, null=True, db_index=True)
+
+    class Meta:
+        db_table = "emails_profile"
```

Comment:
(suggestion, non-blocking) Add comment here that also points to migrations used and documentation on how the model was moved from `emails/models.py`